functions/routes/bling/callback.js

```diff
@@ -29,7 +29,7 @@ exports.post = ({ appSdk, admin }, req, res) => {
       const clientIp = req.get('x-forwarded-for') || req.connection.remoteAddress
       */
       res.sendStatus(200)
-
+      console.log(storeId)
```

Comment:
Pra quê esse log aí? E se realmente for pra adicionar um log, deveria ser um melhor, não?

Reply:
Quando dá erro de autenticação o `storeId` tá no objeto do erro que vai pro log https://github.com/ecomplus/app-bling-erp/blob/master/functions/routes/bling/callback.js#L203-L204